packages/opentelemetry-tracing/test/Tracer.test.ts

```diff
@@ -220,4 +221,37 @@ describe('Tracer', () => {
     assert.strictEqual(context.traceFlags, TraceFlags.NONE);
     span.end();
   });
+
+  it('should start an active span', () => {
+    const tracer = new Tracer(
+      { name: 'default', version: '0.0.1' },
+      { sampler: new TestSampler() },
+      tracerProvider
+    );
+
+    assert.strictEqual( tracer.startActiveSpan('my-span', span => {
```

Comment:
Please add an assertion that span is active

`assert.strictEqual(span, context.active())`

Reply:
`assert.strictEqual(getSpan(context.active()), span)` fails in all cases because `getSpan(context.active())` is undefined, will investigate... how can I know which context manager is set? If it is Noop this may be expected as per getSpan(context.active())

Reply:
Ah yeah you probably want to set a context manager in the `before` block of the tests (and call `.reset()` in the `after` block.

Reply:
this functionality is now tested in node and web packages as recommended here: https://github.com/open-telemetry/opentelemetry-js/pull/2221#pullrequestreview-665881460

Did you still want to import a context manager here to test this?